lib/NormalModule.js

```diff
@@ -84,7 +84,7 @@ class NormalModule extends Module {
 		this.request = request;
 		this.userRequest = userRequest;
 		this.rawRequest = rawRequest;
-		this.binary = type.startsWith("webassembly");
+		this.binary = /(url|webassembly)/.test(type);
```

Comment:
What about something like
```js
/^(url|webassembly)\b/
```
?

Reply:
We only have e.g `url` (future) and `url/experimental` as valid `module.type` the `/` is kind of 'enforced' here to specify the subtype (if any)